### PR TITLE
Language server

### PR DESCRIPTION
**Pull request by jdoshi8**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,8 +39,8 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
 lib64/
+lib/
```
Comment (Anubhav2701) [inline .gitignore]: remove this change. lib should move back to its original line number.

File: airflow/www_rbac/views.py [hunk 2]
```diff
@@ -4332,6 +4339,7 @@ def editdag(self, filename):
         return self.render_template("airflow/editdag.html",
                                     code=code,
                                     filename=filename,
+									dags_folder_path=settings.DAGS_FOLDER,
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: Indentation

**Comment by Anubhav2701** [comment]
Can you convert `serverURI` in `langserver.js` to a generic URI? `ws://localhost:3000/python` will not work when client is accessing it over a network. It might need you to fork the `lsp-adapter` for codemirror.., I am not sure.. ?

Make it configurable via env variables from airflow ? I mean, keep the uri in format `ws://<ip or url>/langserver/python` where  `/langserver/python` is configurable from airflow and you can extract `<ip or url>` from the browser.

**Comment by jdoshi8** [comment]
> Can you convert `serverURI` in `langserver.js` to a generic URI? `ws://localhost:3000/python` will not work when client is accessing it over a network. It might need you to fork the `lsp-adapter` for codemirror.., I am not sure.. ?
> 
> Make it configurable via env variables from airflow ? I mean, keep the uri in format `ws://<ip or url>/langserver/python` where `/langserver/python` is configurable from airflow and you can extract `<ip or url>` from the browser.

I can change the serverURI directly in the file. However the python-jsonrpc-server library that provides the WebSocket will also have to be modified. After that it cannot be installed directly.

**Comment by Anubhav2701** [comment]
I think changing `langserver_ext.py` would work.
Also, keep langserver_ext.py in  `airflow/www_rbac/langserver_ext.py`, not in `airflow/www_rbac/static/`

File: airflow/settings.py [hunk 3]
```diff
@@ -244,6 +245,7 @@ def configure_vars():
     MODEL_SERVERS = normalize_path(os.path.join(AIRFLOW_HOME, *[os.pardir, 'trained-models']))
     LIVY_CONF_PATH = deepcopy(HADOOP_CONFIGS_FOLDER)
     CHANGELOG_PATH = os.path.join((os.path.abspath(os.path.dirname(__file__))), 'changelog.yaml')    
+	LANGUAGE_SERVER_PATH = "/langserver/python"
```
Comment (Anubhav2701) [inline airflow/settings.py]: Read from the env variables, if not present, then use `"/langserver/python"` as default value.
Something like:
LANGUAGE_SERVER_PATH = os.environ.get('LANGUAGE_SERVER_PATH', "/langserver/python")

Comment (Anubhav2701) [reply airflow/settings.py]: AIso, incorrect indentation

Comment (jdoshi8) [reply airflow/settings.py]: Okay

**Pull request closed without merging**